settings.gradle

```diff
@@ -1,6 +1 @@
-rootProject.name = 'ably-java'
-include 'lib'
-
-if(System.getenv('ANDROID_HOME')) {
-  include 'android-test'
-}
+include ':core', ':android', ':java'
```

Comment:
I wanted to go with artifact prefix as a module name after all.
  - `core` for `ably-core`
  - `java` for `ably-java`
  - `android` for `ably-android`